python/vision_explanation_methods/error_labeling/example_notebook_error_labeling.ipynb

```diff
@@ -0,0 +1,279 @@
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from responsibleai_vision import ModelTask, RAIVisionInsights\n",
+    "import os\n",
+    "import sys\n",
+    "from zipfile import ZipFile\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import datasets\n",
+    "import matplotlib\n",
+    "matplotlib.use('Agg')\n",
+    "import matplotlib.pyplot as plt\n",
+    "from responsibleai_vision.common.constants import ImageColumns\n",
+    "import json\n",
+    "from fastai.learner import load_learner\n",
+    "from raiutils.common.retries import retry_function\n",
+    "from ml_wrappers.model.image_model_wrapper import PytorchDRiseWrapper\n",
+    "\n",
+    "import json\n",
+    "import os\n",
+    "import xml.etree.ElementTree as ET\n",
+    "import torchvision.datasets as dset\n",
+    "\n",
+    "import torchvision\n",
+    "from torchvision.models.detection.faster_rcnn import FastRCNNPredictor\n",
+    "import torch\n",
+    "import os\n",
+    "\n",
+    "\n",
+    "try:\n",
+    "    from urllib import urlretrieve\n",
+    "except ImportError:\n",
+    "    from urllib.request import urlretrieve\n",
+    "\n",
+    "import torchvision\n",
+    "from torchvision.models.detection.faster_rcnn import FastRCNNPredictor\n",
+    "import torch\n",
+    "import os\n",
+    "\n",
+    "from PIL import Image\n",
+    "from torchvision import transforms as T\n",
+    "from vision_explanation_methods.error_labeling.error_labeling import (\n",
+    "    ErrorLabeling, ErrorLabelType)\n",
+    "from ml_wrappers.model.image_model_wrapper import WrappedObjectDetectionModel\n",
+    "from IPython.display import display\n",
+    "from PIL import Image, ImageDraw\n",
+    "import copy\n",
+    "from PIL import Image, ImageDraw, ImageFont"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\n",
+    "def load_mscoco_object_detection_dataset_labels():\n",
+    "\n",
+    "    src_images = \"./dataCoco/images/\"\n",
+    "    base_image = './dataCoco/'\n",
+    "\n",
+    "    # Path to the annotations\n",
+    "    annotations_folder = os.path.join(base_image, \"annotations\")\n",
+    "    \n",
+    "    path2data = './dataCoco/val2017/'\n",
+    "    path2json = './dataCoco/annotations/instances_val2017.json'\n",
+    "    coco_val = dset.CocoDetection(root=path2data, annFile=path2json)\n",
+    "\n",
+    "    image_labels = []\n",
+    "    image_ids = []\n",
+    "    for x in range(4,10):\n",
+    "        img, target = coco_val[x]\n",
+    "        print(img)\n",
+    "        print(target[0]['image_id'])\n",
+    "        img_label = []\n",
+    "        image_ids.append(str(target[0]['image_id']))\n",
+    "        for i in range(0, len(target)):\n",
+    "            \n",
+    "            #sprint(target[i])\n",
+    "            bbox = target[i]['bbox']\n",
+    "            print(bbox)\n",
```

Comment:
I think printing every image and bounding box might be a bit excessive for a notebook example?